test/unit/core/simd/view_iota_simd_test.cpp

```diff
@@ -0,0 +1,74 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include <seqan3/core/simd/simd.hpp>
+#include <seqan3/core/simd/view_iota_simd.hpp>
+#include <seqan3/range/container/aligned_allocator.hpp>
+#include <seqan3/test/simd_utility.hpp>
+
+#include "../../range/iterator_test_template.hpp"
+
+template <typename simd_t>
+struct iota_simd_test : public ::testing::Test
+{};
+
+template <typename simd_t>
+struct iterator_fixture<iota_simd_test<simd_t>> : public iota_simd_test<simd_t>
```

Comment:
:laughing: 

How you want this style so badly that you do it here, but one part of the team likes the "setting"-something-globally-style better xD